addons/web/static/lib/bootstrap/scss/_variables.scss

```diff
@@ -1748,4 +1748,7 @@ $nested-kbd-font-weight:            null !default; // Deprecated in v5.2.0, remo
 
 $pre-color:                         null !default;
 
+// align (Search...) with long input values.
+$search-align-width: 16rem;
+
```

Comment:
:warning:  **Do not, under any circumstances, modify the vendored libraries!** :warning: 

There are dedicated file for defining the variables...

Reply:
I have defined the variables in primary_variables.scss instead of creating a new file, search_bar.variables.scss, in the stable version.

Reply:
I have added a new file named `search_bar.variables.scss`.